internal/services/attestation/attestation_provider_resource.go

```diff
@@ -169,13 +221,35 @@ func resourceAttestationProviderUpdate(d *pluginsdk.ResourceData, meta interface
 		return err
 	}
 
+	var hasChange bool
 	updateParams := attestationproviders.AttestationServicePatchParams{}
 	if d.HasChange("tags") {
+		hasChange = true
 		updateParams.Tags = tags.Expand(d.Get("tags").(map[string]interface{}))
 	}
 
-	if _, err := client.Update(ctx, *id, updateParams); err != nil {
-		return fmt.Errorf("updating %s: %+v", *id, err)
+	if hasChange {
+		if _, err := client.Update(ctx, *id, updateParams); err != nil {
+			return fmt.Errorf("updating %s: %+v", *id, err)
+		}
+	}
+
+	if d.HasChange("policy") {
+		policies, err := expandPolicies(d.Get("policy").([]interface{}))
+		if err != nil {
+			log.Printf("[Warn] expand policies: %+v", err)
+		}
+		policyClient := meta.(*clients.Client).Attestation.PolicyClient
+		url := d.Get("attestation_uri").(string)
+		if url == "" {
```

Comment:
white space for readability
```suggestion
		}
		policyClient := meta.(*clients.Client).Attestation.PolicyClient
		
		url := d.Get("attestation_uri").(string)
		if url == "" {
```